drivers/scheduler/k8s/specs/wordpress-fb-nfs4/storage.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: wordpress-sc-fb-nfs4
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  backend: "pure_file"
+mountOptions:
+- nfsvers=4.1
```

Comment:
@pure-jliao Is this always needed? Do we need to test for other versions as well ?

Reply:
right now only support `nfsvers=3` and `nfsvers=4.1`, we could leave `mountOptions` empty to use v3 by default.

Reply:
This spec is specifically called `wordpress-fb-nfs4`, so its ok to leave the mount option, the default `wordpress` spec will use the default nfs3 as there is no `mountOptions` specified.